app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    source_file = command[1]
+    destination_file = command[2]
+    if len(command) != 2:
```

Comment:
<img width="641" alt="image" src="https://github.com/mate-academy/py-copy-file/assets/65166203/c319eb21-b349-45df-899a-3b6480d8a1ed">
First of all. Command is always 3 params
Also your code will not work if it comes with 2 params because you do [2]
Check that command is == "cp"